.github/inactive-workflows/build_scoped_rector_php70.yaml

```diff
@@ -73,18 +95,17 @@ jobs:
                     php-version: 7.0
                     coverage: none
             -   run: |
-                    rm -rf vendor && mkdir -p vendor
-                    cp -R rector-prefixed-downgraded-php70/vendor/* vendor/
-                    rm -rf php-parallel-lint
+                    rm -rf vendor
```

Comment:
vendor is removed first due can't install parallel lint without removing vendor first.